obspy/core/tests/test_response.py

```diff
@@ -483,6 +484,52 @@ def test_regression_evalresp(self):
              2.445572e+05 - 2.480459e+03j,
              -2.455459e-01 + 4.888214e-02j], rtol=1e-6)
 
+    def test_get_flat_response_without_poles_and_zeros(self):
+        """
+        Tests the get_flat_response calss method without poles and zeros.
+        """
+
+        resp = Response.get_flat_response()
```

Comment:
Can you test for some other sensitivity than 1? Testing with 1 sometimes hides some issues as multiplication/division are the same with it.

Reply:
Good call.  I switched the test cases to the value 5.